vector/src/main/res/drawable/ic_close_with_circular_bg.xml

```diff
@@ -0,0 +1,23 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+    <path
+        android:fillAlpha="0.9"
+        android:fillColor="#E3E8F0"
+        android:pathData="M24,12C24,18.6274 18.6274,24 12,24C5.3726,24 0,18.6274 0,12C0,5.3726 5.3726,0 12,0C18.6274,0 24,5.3726 24,12Z"
+        android:strokeAlpha="0.9" />
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M7.9998,7.9998L15.9998,15.9998"
+        android:strokeWidth="1.33333"
+        android:strokeColor="#737D8C"
+        android:strokeLineCap="round" />
+    <path
+        android:fillColor="#00000000"
+        android:pathData="M16.0005,7.9998L8.0006,15.9998"
+        android:strokeWidth="1.33333"
+        android:strokeColor="#737D8C"
```

Comment:
this asset uses hardcoded colours as values are the same for both light and dark

Reply:
Maybe this is an error on the Figma, did you double check with @amshakal ?

Reply:
will raise in the daily sync :+1:

Reply:
great catch, updated the screenshots in the PR description https://github.com/vector-im/element-android/pull/4610/commits/ca8ac1c1adaedd793198f2dd313cda87125115dc

Reply:
Cool, thanks